FF153 IDBIndex/IDBObjectStore.getAllRecords() (#44510)
mdn/content: files/en-us/mozilla/firefox/releases/153/index.md

### APIs

- The {{domxref("IDBObjectStore.getAllRecords()")}} and {{domxref("IDBIndex.getAllRecords()")}} methods are now supported.

>   These retrieve all records (or some some specified subset of records) from an IDB object store and index, respectively.

Reviewer: just few nits
```suggestion
  These retrieve all records (or a specified subset of records) from an object store and index, respectively.
```
